priv/repo/migrations/20200904184516_alter_sessions.exs

```diff
@@ -0,0 +1,18 @@
+defmodule Hexpm.RepoBase.Migrations.AlterSessions do
+  use Ecto.Migration
+
+  def change do
+    alter table(:sessions) do
+      add(:user_id, references(:users))
+      add(:uuid, :uuid, unique: true)
+      add(:expires_at, :utc_datetime_usec)
+      add(:active, :boolean, default: false)
+    end
+
+    rename(table(:sessions), :token, to: :token_hash)
+    create(unique_index(:sessions, [:user_id, :uuid, :token_hash]))
```

Comment:
I can't find where we look up on this. Maybe only keep `[:user_id]` or `[:user_id, :uuid]`?

Reply:
Yeah, I added it as a constraint (since ecto doesn't support unique multiple column constraints in the DSL yet AFAIK), but it's probably overkill, so removed.